bn-apps/memberships-management/membership-service/src/main/kotlin/net/corda/businessnetworks/membership/member/RequestMembershipFlow.kt

```diff
@@ -17,8 +22,7 @@ data class OnBoardingRequest(val metadata : Any)
  */
 @StartableByRPC
 @InitiatingFlow
-class RequestMembershipFlow(private val membershipMetadata : Any) : FlowLogic<SignedTransaction>() {
-
+class RequestMembershipFlow(bno : Party, private val membershipMetadata : Any) : BusinessNetworkAwareInitiatingFlow<SignedTransaction>(bno) {
```

Comment:
Is it possible to ensure that the bno parameter is BNO, if so it should be done.

Reply:
Please have a look at `BusinessNetworkAwareInitiatingFlow`